### DISCUSSION TITLE
[Feature] Delete Photos on local device to save storage

### DISCUSSION THREAD

**Discussion by lx05** [immich-app/immich#14323] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would love to see a feature where Immich automatically deletes the local files on my phone and only uses the server version to save Storage.

There is a similar function in G Photos where you can delete the already backed up files on the device if older than x. I would love Immich to do that, right now I have to check if the files are uploaded and then delete them manually via a file explorer as described here ([https://github.com/immich-app/immich/discussions/2379]).

The function does not have to run automatically, I just want to click a button if I have low free storage space and Immich tells me something like:

"All Media that is backed up and older than 1 month will be deleted from the local device to save storage. This will delete "AmountX" Pictures and "AmountY" Videos and free up "AmountZ" GB of storage. 
Do you want to proceed"

I also attached a screenshot, but it is german.

![Screenshot_20241125-001740](https://github.com/user-attachments/assets/0ab1edb6-87d8-4b67-8d31-5b0f81060e6e)

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
duplicate of #165, this will be added in/after stable release. thank you
